pm.mirko.Amusiz.yml

```diff
@@ -0,0 +1,22 @@
+id: pm.mirko.Amusiz
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: '40'
+command: amusiz
+finish-args:
+- "--allow=multiarch"
+- "--share=network"
+- "--share=ipc"
+- "--socket=fallback-x11"
+- "--socket=pulseaudio"
+- "--socket=wayland"
+- "--system-talk-name=org.freedesktop.GeoClue2"
```

Comment:
Same question. (As geolocation in music app seems odd without context.)

Reply:
I used the manifest from another application of mine and forgot to remove it